thorlcr/graph/thgraph.hpp

```diff
@@ -583,9 +584,14 @@ class graph_decl CGraphBase : public CInterface, implements IEclGraphResults, im
     CGraphBase *queryOwner() { return owner; }
     CGraphBase *queryParent() { return parent?parent:this; }
     IMPServer &queryMPServer() const;
+    void clearProgressUpdated() { progressUpdated.store(false); }
+    void setProgressUpdated() { progressUpdated.store(true); }
+    bool hasProgressUpdated()
```

Comment:
minor: it isn't clear from the function name that it has the side-effect of clearing the current setting.